util/container/README.md

```diff
@@ -10,7 +10,7 @@ There is a pre-built version of the container available online. This version is
 
 To download the container, first login to the GitHub container registry:
 ```shell
-$ docker login ghcr.io
+$ docker login docker.io
 ```
 You will be asked for a username (your GitHub username).
 As a password you should use a
```

Comment:
You can not login to the dockerhub registry with a github PAT.
Please adapt these lines.